priv/test_files/ignore/specific_ignore.erl

```diff
@@ -0,0 +1,9 @@
+-module(no_ignore).
```

Comment:
Just a detail, but it should be renamed to `specific_ignore`, right?

Reply:
🤦